starlette/websockets.py

```diff
@@ -181,6 +202,12 @@ async def close(
             {"type": "websocket.close", "code": code, "reason": reason or ""}
         )
 
+    async def send_response(self, response: Response) -> None:
+        if self._have_response_extension():
+            await response(self.scope, self.receive, self.send)
+        else:
+            await self.close(code=1008, reason=f"HTTP Response {response.status_code}")
```

Comment:
```suggestion
        raise RuntimeError(""The server doesn't support the Websocket Denial Response extension.")
```

Reply:
See above.  I don't think we should bounce an exception on the developer when a purely informational message is not deliverable because of a choice of web server.  But I suggest that we add a warning instead?